### DISCUSSION TITLE
5.11 新增加的table ref 怎么使用的，有没有demo，一升级搞的我之前加的所有的useref的值都不能用了，页面直接报错

### DISCUSSION THREAD

**Discussion by guanshining** [ant-design/ant-design#45984] [Q&A] [answered]
5.11 新增加的table ref 怎么使用的，有没有demo，一升级搞的我之前加的所有的useref的值都不能用了，页面直接报错，快来个demo

**Comment by zombieJ** [comment, marked as answer]
报错信息是什么能看看不？

tableRef 说明在这里：https://ant-design.antgroup.com/components/table-cn#table-ref

用法就是：
```tsx
const tableRef = React.useRef<TableRef>();

<Table ref={tableRef} />

tableRef.current.nativeElement // Table DOM
tableRef.current.scrollTo(...) // 滚动到目标位置

```

**Reply by mrleidesen** [reply]
`const tableRef = React.useRef<TableRef>();` -> `const tableRef = React.useRef<TableRef>(null);`

**Reply by zhimng** [reply]
TableRef 是从哪里引入的呢
